pytest_splunk_addon/splunk.py

```diff
@@ -293,6 +299,26 @@ def splunk_search_util(splunk, splunk_setup, request):
 
     return search_util
 
+@pytest.fixture(scope="session")
+def suppress_internal_errors(request):
+    """
+    This fixture generates a common list of errors which are suppossed 
+    to be ignored in test_splunk_internal_errors.
+
+    Returns:
+        dict: List of the strings to be ignored in test_splunk_internal_errors
+    """
+    error_list = []
+    splunk_error_file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)),".ignore_splunk_internal_errors")
+    with open(splunk_error_file_path) as splunk_errors:
```

Comment:
This block can be moved inside the if because is option is not specified, there is no need to read a file.

Reply:
The idea is to maintain two files one containing only addon related errors for which an option is provided and another one for Splunk related errors which is maintained in pytest-splunk-addon itself.

Reply:
Got it, thanks!